homeassistant/config.py

```diff
@@ -117,6 +118,26 @@
 """
 
 
+@dataclass
+class ConfigExceptionInfo:
+    """Configuration exception info class."""
+
+    exception: Exception
+    translation_key: str
+    platform_name: str
+    config: ConfigType
+    integration_link: str | None = None
+    notify: bool = False
```

Comment:
Please remove this, setup.py + bootstrap.py should check the translation key to decide if they want to notify or now